src/packages/utils.ts

```diff
@@ -32,7 +32,7 @@ const translateRequestOptionsToFetchOptions = (options: any): { url: string, fet
         url += options.url;
     }
     if (options.qs) {
-        url += options.qs;
+        url += "?" + Object.values(options.qs).toString();
```

Comment:
serialize = function(obj) {
  var str = [];
  for (var p in obj)
    if (obj.hasOwnProperty(p)) {
      str.push(p + "=" + obj[p]);
    }
  return str.join("&");
}